tests/test_smoke.py

```diff
@@ -3225,7 +3225,7 @@ def test_nonexistent_bucket(self, nonexist_bucket_url):
 
         with pytest.raises(
                 sky.exceptions.StorageBucketGetError,
-                match='Attempted to connect to a non-existent bucket'):
```

Comment:
is it possible to add a smoke test like `test_kubernetes/aws/gcp_storage_mounts`, except changed to `test_storage_mounts_with_stop` to test the stop start functionality? Basically:
1. Launch `tests/test_yamls/test_storage_mounting.yaml.j2` on `generic_cloud`
2. Stop the cluster
3. Start again
4. See if mounting works succesfully by both reading and writing files to `MOUNT` paths.